### DISCUSSION TITLE
Ignore/Filter shows that don't interest me

### DISCUSSION THREAD

**Discussion by jeidon** [jellyfin/jellyfin#10500] [General] [answered]
I have a medium-sized collection of shows and as they are imported, they show in the "Latest Shows" section.

Several of the shows are things my family members have requested that I don't care about.

While it is only a slight annoyance, I'd love to be able to mark a show as "Uninterested" and perhaps filter them from my normal views and only see them if I really wanted to.

I'd be willing to work on a plug-in for this but need some help getting started. I am a decent developer but can't seem to find the C# documentation that would help me get going.

I'm open to all feedback on how I could do this or how I could get started developing a plug-in.

As an FYI, I have already downloaded the jellyfin-plugin-template and have begun poking around in there but I imagine there's a bit of work to do both on the client side and the database side to support this sort of request.

Thanks

**Comment by crobibero** [comment, marked as answer]
Plugins don’t have the ability to do what you want. 

One thing you might be able to do is block items with a specific tag, and then add the tag for the media you don’t want to see

**Reply by jeidon** [reply]
Oh, bummer, I was hoping there'd be something I could do but like I mentioned, not a huge deal just sort of visual noise.

I'll give the tags a shot and see how it works out.

Thanks
